apps/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       - db_nw
       - web_nw
     env_file:
-      - ./.docker/web.env
+      - ./api/.docker/web.env
```

Comment:
`.docker/web.env` should be located in `apps/.docker/web.env` and not `apps/api/.docker/web.env` as I moved that file as well unless I missed something

Reply:
unless we move the `.web.env.default` as well

Reply:
You had the `web.env`, which only has information required for the `api` in the `apps` folder, so I moved it to the `api` folder, along with the `web.env.default`. Please clarify concern

Reply:
I missed the top file, you're good to go